### DISCUSSION TITLE
Can the vomnibar be made more fuzzy?

### DISCUSSION THREAD

**Discussion by txtyash** [gdh1995/vimium-c#864] [Ideas]
The current search in vomnibar isn't as flexible as something like [fzf](https://github.com/junegunn/fzf). The spaces are a compulsion and sequencing is strict. For example let's say I want to go to my previous search of `exercism.org` so I type `exm` then I would get no matches.

A good fuzzy search would save a lot of time & effort here.

*vimiumC is great either ways. Thanks for putting your time in maintaining it*

**Comment by gdh1995** [comment]
Um I think fuzzy matching may be somehow slow, at least when code is written in JavaScript, so I have no plan to implement it.
